src/Components/ToDoList/ToDoList.tsx

```diff
@@ -1,20 +1,14 @@
-import React, { ReactNode, useEffect } from "react";
+import React, { ReactNode } from "react";
 import TodoItem from "components/TodoItem/TodoItem";
 
-import { useDispatch, useSelector } from "react-redux";
+import { useSelector } from "react-redux";
 import { AppState } from "store/store";
-import { fetchTodos } from "store/todo/todoActions";
 
 const TodoList = () => {
-  const dispatch = useDispatch();
   const { todos } = useSelector((state: AppState) => state.todosReducer);
 
-  useEffect(() => {
-    dispatch(fetchTodos());
-  }, [dispatch]);
-
   const renderToDoList = (): ReactNode =>
-    todos.map((todo) => <TodoItem key={todo.id} todo={todo} />);
+    todos.map((todo, index) => <TodoItem key={index} todo={todo} />);
```

Comment:
If todos have an id field it is better to use todo id instead of index but if not then it's okay

Reply:
There was a problem with ids, for now it is on purpose